src/matlab2tikz.m

```diff
@@ -98,6 +98,9 @@ function matlab2tikz(varargin)
 %   interpret tick labels as TeX. MATLAB(R) doesn't do that by default.
 %   (default: false)
 %
+%   MATLAB2TIKZ('arrowHeadSizeFactor', FLOAT, ...) allows to resize the arrow heads
+%   in quiver plots by rescaling the arrow heads by a positive scalar. (default: 10)
```

Comment:
It's still a factor (i.e. it scales the length), but fair enough to change it since that monstrosity is way too long anyway (and factor says nothing important to the user).